vscode/src/chat/chat-view/ChatController.ts

```diff
@@ -519,9 +502,9 @@ export class ChatController implements vscode.Disposable, vscode.WebviewViewProv
                     break
                 }
                 if (message.authKind === 'signin' && message.endpoint && message.value) {
-                    await authProvider.auth({
-                        endpoint: message.endpoint,
-                        token: message.value,
+                    await localStorage.saveEndpointAndToken({
+                        serverEndpoint: message.endpoint,
+                        accessToken: message.value,
```

Comment:
```suggestion
                    await authProvider.validateAndStoreCredentials(
                        {
                            serverEndpoint: message.endpoint,
                            accessToken: message.value,
                        },
                        'store-if-valid'
                    )
```

Should we use `validateAndStoreCredentials` here when user tries to sign in via the UI?

Reply:
In this case, no, since I think the desired behavior is for those creds to be stored no matter what, even if the user is (for example) temporarily offline and the creds can't be validated. I could see it going either way.